sdk-network-providers/tokens.md

```diff
@@ -0,0 +1,24 @@
+class FungibleTokenOfAccountOnNetwork:
+    identifier: str
+    balance: Amount
+    rawResponse: Any
```

Comment:
When you read proxy you get only identifier+balance for token, but with API you get a lot more important info (decimals, ticker, owner, assets etc.). Need a way (maybe another call) to get more info about the token so it can actually be used at least for displaying balance.